src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -195,7 +195,8 @@ private static DistroInfo GetDistroInfo()
                 // example:
                 //   SunOS 5.11 11.3
                 result.Id = "Solaris";
-                result.VersionId = ToVersion(RuntimeInformation.OSDescription.Split(' ')[2]); // e.g. 11.3
+                // we only need the major version; 11
```

Comment:
Why strip it off here rather than making it available?  Consumers can pay attention to just the major version if they want.

Reply:
This was to align the version scheme used by installer and SDK for Solaris (similar to FreeBSD case above): https://github.com/dotnet/runtime/blob/ea06e0c05c34602d4fbb920cff30f681f30b8d81/src/installer/corehost/cli/hostmisc/pal.unix.cpp#L676-L677 https://github.com/dotnet/sdk/blob/c4d2c8c303b9a7eb09e032337b7649556b777a07/src/Cli/Microsoft.DotNet.Cli.Utils/RuntimeEnvironment.cs#L70-L73.

Reply:
In that case it's probably better to say "want" rather than "need"?